.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+
+updates:
+  - package-ecosystem: 'github-actions'
+    directory: '/'
+    schedule:
+      interval: 'weekly'
```

Comment:
Interesting, why weekly and not daily? 🤔

Reply:
I feel like we don't care much about having actions updated, also if they would require workflow changes - less likely we will need to address ASAP. If to dedicate some time once a week (if any), IMO is enough?

P.S. We can also see how often such occur and change the preference, if we see we'd need to.